helm-charts/packit-service-validation/templates/job-run-validation.yml

```
@@ -1,36 +1,57 @@
 # https://docs.openshift.com/container-platform/latest/rest_api/workloads_apis/cronjob-batch-v1.html
+{{- range $cronjob := $.Values.cronjobs }}
 apiVersion: batch/v1
 kind: CronJob
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ $.Release.Name }}-{{ $cronjob.id }}
 spec:
-  schedule: {{ .Values.cronjob.schedule | quote }}
+  schedule: {{ $cronjob.schedule | quote }}
   jobTemplate:
     spec:
-      activeDeadlineSeconds: {{ .Values.cronjob.activeDeadlineSeconds }}
+      activeDeadlineSeconds: {{ $cronjob.activeDeadlineSeconds }}
       template:
         spec:
           containers:
           - name: run-validation
-            image: "{{ .Values.image.repo }}:{{ .Values.image.tag }}"
+            image: "{{ $.Values.image.repo }}:{{ $.Values.image.tag }}"
             env:
+{{ if eq $cronjob.id "daily-validation" }}
             - name: GITHUB_TOKEN
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Release.Name }}-github
+                  name: {{ $.Release.Name }}-github
                   key: token
             - name: GITLAB_TOKEN
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Release.Name }}-gitlab
+                  name: {{ $.Release.Name }}-gitlab
                   key: token
+{{ end }}
+{{ if eq $cronjob.id "weekly-validation"}}
+            - name: GITLAB_GNOME_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $.Release.Name }}-gitlabgnome
+                  key: token
+            - name: GITLAB_FREEDESKTOP_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $.Release.Name }}-gitlabfreedesktop
+                  key: token
+            - name: SALSA_DEBIAN_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $.Release.Name }}-salsadebian
+                  key: token
+{{ end }}
```

I've just tried to run `make template foobar` in `values/packit-service-validation/` and there are a few empty lines in the output, which I think shouldn't be a problem (as it's still a valid YAML), but can be improved:

```suggestion
{{- end }}
{{- if eq $cronjob.id "weekly-validation"}}
            - name: GITLAB_GNOME_TOKEN
              valueFrom:
                secretKeyRef:
                  name: {{ $.Release.Name }}-gitlabgnome
                  key: token
            - name: GITLAB_FREEDESKTOP_TOKEN
              valueFrom:
                secretKeyRef:
                  name: {{ $.Release.Name }}-gitlabfreedesktop
                  key: token
            - name: SALSA_DEBIAN_TOKEN
              valueFrom:
                secretKeyRef:
                  name: {{ $.Release.Name }}-salsadebian
                  key: token
{{- end }}
```